packer/sagebio/rStudioAmi-LATEST/playbook.yaml

```diff
@@ -0,0 +1,53 @@
+- hosts: all
+  become: yes
+
+  tasks:
+    - name: install system updates for centos systems
+      yum: name=* state=present update_cache=yes
+      when: ansible_distribution == "CentOS"
+
+    - name: install system updates for ubuntu systems
+      apt: upgrade=dist update_cache=yes
+      when: ansible_distribution == "Ubuntu"
+
+    - name: install packages
+      package:
+        name:
+          - "ntpdate"
+          - "curl"
+          - "dpkg-dev"
+          - "zlib1g-dev"
+          - "libssl-dev"
```

Comment:
I think this might be a problem, libssl update causing an interactive prompt
https://bugs.launchpad.net/ubuntu/+source/ansible/+bug/1833013

please use this workaround, https://github.com/Sage-Bionetworks/ami-provisioner/blob/master/packer/weechat/weechat-LATEST/bootstrap.sh